yotiattributevalue.go

```diff
@@ -39,11 +42,14 @@ func (val AttributeValue) GetContentType() (result string) {
 	case AttributeTypeText:
 		result = "text/plain; charset=UTF-8"
 
-	case AttributeTypeJpeg:
+	case AttributeTypeJPEG:
 		result = "image/jpeg"
 
-	case AttributeTypePng:
+	case AttributeTypePNG:
 		result = "image/png"
+
+	case AttributeTypeJSON:
+		result = "application/json; charset=UTF-8"
```

Comment:
shouldn't switch _always_ have a default case? even if it's just for logging an error?

Reply:
Yep good spot, will add